Content.Server/Speech/EntitySystems/SkeletonAccentSystem.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Text.RegularExpressions;
+using Content.Server.Speech.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.Speech.EntitySystems;
+
+public sealed class SkeletonAccentSystem : EntitySystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    private static readonly Dictionary<string, string> DirectReplacements = new()
+    {
+        { "fuck you", "I've got a BONE to pick with you" },
+        { "fucked", "boned"},
+        { "fuck", "RATTLE RATTLE" },
+        { "fck", "RATTLE RATTLE" },
+        { "shit", "RATTLE RATTLE" }, // Capitalize RATTLE RATTLE regardless of original message case.
+        { "definitely", "make no bones about it" },
+        { "absolutely", "make no bones about it" },
+        { "lonely", "bonely"},
```

Comment:
this is already handled by the -one regex

Reply:
good catch - fixed.